stablehlo/reference/Ops.cpp

```diff
@@ -1,4 +1,4 @@
-/* Copyright 2023 The StableHLO Authors.
+/* Copyright 2024 The StableHLO Authors.
```

Comment:
With a bit of digging: I think the year here when the material was published unless we have a range of years (e.g., 2023–2024), starting from the date of the oldest published  and ending with the date of the newest updates. 
Should be update that?